tests/voq/test_voq_nbr.py

```diff
@@ -614,7 +614,8 @@ def test_neighbor_clear_one(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
         check_neighbors_are_gone(duthosts, all_cfg_facts, per_host, asic, nbr_to_test)
 
         logger.info("Verify other neighbors are not affected.")
-        dump_and_verify_neighbors_on_asic(duthosts, per_host, asic, untouched_nbrs, nbrhosts, all_cfg_facts, nbr_macs)
+        dump_and_verify_neighbors_on_asic(duthosts, per_host, asic, untouched_nbrs,
+                                          nbrhosts, all_cfg_facts, nbr_macs, check_nbr_state=False)
```

Comment:
@sanjair-git is this behavior of one of the untouched neighbor going stale is observed always -- if so good to find the reason ?

Reply:
Hi @judyjoseph,

For fully loaded T2 chassis, YES! most of the times. Because,

VOQ-Nbr tests disables BGP at the start of test suite. For multi-asic T2 chassis, this test selects random 4 neighbors on one of the asic of a line card for neighbor flush and remaining 22 neighbors as untouched on the same asic. Once the four neighbors are cleared, test checks for the following things and it nearly takes 4 minutes for fully loaded _100/400G T2-max,_

- Poll for arp clear and make sure all of those 4 neighbors are deleted from the arp table on all 4 asics of two line cards. 
- Do sonic-db-dump on _ASIC_DB_, _APPL_DB_ and _CHASSIS_APP_DB_ for each asic and verify all those 4 neighbors are removed.
- Verify bgp and host kernel routes are removed for these neighbors on each asic.

Hence after ~4 minutes, when the test comes to the verification of these 22 untouched neighbors which runs close to another ~15+ minutes, since there was no traffic with bgp disabled, few of the neighbors' state tend to go to STALE.

Reply:
Sure, Can we set the check_nbr_state = True here --> https://github.com/sonic-net/sonic-mgmt/blob/4d9902c8dc5e484bcd65d6a48dfb74f3411c0f0e/tests/voq/test_voq_nbr.py#L630

Here the neighbors should be in good state

Reply:
Hi Judy,

Thank you. Last year, looks Sandeep purposefully added this check with the PR [4953](https://github.com/sonic-net/sonic-mgmt/pull/4953) during relearn of the neighbors, by mentioning the following,

```
   - test_neighbor_clear_one, test_neighbor_clear_all:
       In the voq_nbr tests, we disable all eBGP peers on the DUT and the eos peers to avoid any traffic.
       The test does a ping of all the eBGP peers, but in a T2 topology that could take 90+ seconds and then
       checks if the neighbor entries are installed in all asic. With no other traffic, some neighbors might
       not always be in 'REACHABLE' state. So, ignoring the state and just checking that the neighbor entry exists
```
Please take a look.

Reply:
hmm ok. I think we already check this here : https://github.com/sonic-net/sonic-mgmt/blob/master/tests/voq/test_voq_nbr.py#L514

Reply:
Yes Judy, true. We are already checking at #L514.